GoogleDataTransport/GDTCORLibrary/GDTCOREvent.m

```diff
@@ -24,60 +24,32 @@
 
 #import "GDTCORLibrary/Private/GDTCOREvent_Private.h"
 
-static NSString *const kNextEventIDKey = @"GDTCOREventEventIDCounter";
-
 @implementation GDTCOREvent
 
-+ (void)nextEventIDForTarget:(GDTCORTarget)target
-                  onComplete:(void (^)(NSNumber *_Nonnull eventID))onComplete {
-  __block int32_t lastEventID = -1;
-  id<GDTCORStorageProtocol> storage = GDTCORStorageInstanceForTarget(target);
-  [storage libraryDataForKey:kNextEventIDKey
-      onFetchComplete:^(NSData *_Nullable data, NSError *_Nullable getValueError) {
-        if (getValueError != nil || data == nil || data.length == 0) {
-          lastEventID = 1;
-        } else {
-          [data getBytes:(void *)&lastEventID length:sizeof(int32_t)];
-        }
-        if (onComplete) {
-          onComplete(@(lastEventID));
-        }
-      }
-      setNewValue:^NSData *_Nullable(void) {
-        if (lastEventID != -1) {
-          int32_t incrementedValue = lastEventID + 1;
-          return [NSData dataWithBytes:&incrementedValue length:sizeof(int32_t)];
-        }
-        return nil;
-      }];
++ (NSString *)nextEventID {
+  // TODO: Consider a way to make the eventIDs incremental without introducing a storage dependency
```

Comment:
There could be one UUID for each app instance that appends a counter to at least get events from the same app instance incremental.

Reply:
Are UUID's incremental? The reason why I did it the way I did was because I needed a robust mechanism (robust against an app crashing, say) that was consecutive because it would make metalogging in the future easier. Crashlytics may want some metalogging done, so knowing how many events were generated, their order, and what events made it or didn't was valuable.

Reply:
No UUID is not incremental. I suggest to implement the incremental feature separately closer to the time it is required so we don't block the current release.

Reply:
sgtm